features/step_definitions/opportunities_steps.rb

```diff
@@ -15,6 +15,12 @@
   @result_count = page.first(:css, ".search-summary-count").text.to_i
 end
 
+When "I go to $brief URL address" do |brief|
```

Comment:
I think we can use the existing step `Given I go to that brief page` and therefore don't need this new step (although please double check that)